code/modules/projectiles/boxes_magazines/external/pistol.dm

```diff
@@ -83,3 +83,13 @@
 	caliber = ".50"
 	max_ammo = 7
 	multiple_sprites = AMMO_BOX_PER_BULLET
+
+/obj/item/ammo_box/magazine/liberator10mm
+	name = "Makeshift 9mm magazine"
+	desc = "A Makeshift Pistol magazine. Load with 9mm bullets. Usable in Makeshift Pistol."

```

Comment:
```suggestion
	desc = "A handcrafted ammunition magazine, designed to be loaded with 10mm bullets. It seems to be custom-fit for an off-market pistol design."
```
The description of the pistol, and the path name, implies that this is 10mm, not 9mm. Also, we needn't call out the makeshift pistol by name here -- it's not like random spessmen have the time to be writing instructions for use on their illicit handmade firearm equipment.

Reply:
Thanks for fixing this. I wrote the descriptions yesterday when this weapon was still 10mm and I changed it to 9mm this morning forgetting that I had to change the desc too.